discovery-streams/templates/NOTES.txt

```diff
@@ -1,7 +1,7 @@
 1. Get the application URL by running these commands:
-{{- if .Values.ingress.enabled }}
-{{- range .Values.ingress.hosts }}
-  http{{ if $.Values.ingress.tls }}s{{ end }}://{{ . }}{{ $.Values.ingress.path }}
+{{- if .Values.global.ingress.enabled }}
```

Comment:
Good catch. Does this work?

Reply:
I am unsure about this file, this was just so we didn't miss it, I will revert it, try and let you know.

Reply:
After further investigation, this is definitely being recognized and used by helm.. but I don't think it's actually used in the current deployment because we don't have a global TLS value for ingress. We do specify an ssl-policy annotation though which makes me think that maybe we switched how we do ingress TLS at one point?